server/src/DbtProject.ts

```diff
@@ -4,13 +4,26 @@ import { YamlParserUtils } from './YamlParserUtils';
 
 export class DbtProject {
   projectPath: string;
+  parsedProject?: Record<string, unknown>;
+  isDirty = true;
```

Comment:
`isDirty` looks abstract. Maybe make it more explicit like `configWasUpdated`?

Reply:
Renamed.